ui/packages/catalog-ui-search/src/main/webapp/component/query-template-sharing/query-template-sharing.less

```diff
@@ -0,0 +1,111 @@
+@import '../editable-rows/editable-rows';
+
+.icon-label {
+  padding-left: @minimumSpacing;
+}
+
+@{customElementNamespace}template-sharing {
```

Comment:
✏️ `@{customElementNamespace}query-template-sharing` so that it goes along with the filename (and so I can tell from the DOM when I inspect element that if I want to modify `Intrigue-query-template-sharing` I can just go to the `query-template-sharing` component)